common/app/common/templates/HandlebarsTranslationHelper.java

```diff
@@ -14,15 +14,14 @@
 import java.util.*;
 
 import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.toList;
 
 final class HandlebarsTranslationHelper extends Base implements Helper<String> {
```

Comment:
I know it's not about this request, but I would rename this class to CustomI18n, so it is giving a hint that the intention is overriding I18n helper from Hbs Java.